### PR TITLE
jshint cleanup. Add a default global .jshint config. Fix low hanging fruit.

### PR DESCRIPTION
**Pull request by grempe**
I use the jshint plugin in Atom editor. .jshint will work with any jshint
tool though to provide a consistent set of standards for the project.

In this cleanup primarily == to === and !== to !===.

Also fixes some missing (or extraneous) semi-colons and missing use-strict statements.

There are still a number of jshint errors which need to be looked at in
more detail.

See:
http://javascript.crockford.com/style2.html
https://javascriptweblog.wordpress.com/2011/02/07/truth-equality-and-javascript/

All tests (client and server) pass.

### CODE REVIEW

**Comment by daviddahl** [comment]
I like this, will review tomorrow

File: .jshintrc [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+        "superagent",
+        "sjcl",
+        "SRPClient",
+        "isomerize",
+        "self",
+        "async",
+        "BigInteger",
+        "chai"
+    ],
+    "indent"        : 4         // Specify indentation spacing
```
Comment (daviddahl) [inline .jshintrc]: Must be 2 :)

Comment (devgeeks) [inline .jshintrc]: :+1:

File: server/test/account.js [hunk 2]
```diff
@@ -45,7 +47,7 @@ describe('Account model', function () {
       account.update(requestedAccount);
 
       account.save(function (err) {
-        if (err) throw err;
+        if (err) { throw err; }
```
Comment (daviddahl) [inline server/test/account.js]: :+1:

File: server/test/stores/postgres/account.js [hunk 3]
```diff
@@ -16,7 +16,7 @@
  * along with Crypton Server.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-'use strict';
+"use strict";
```
Comment (daviddahl) [inline server/test/stores/postgres/account.js]: dumb question, why the double quotes on all the use stricts?

**Comment by daviddahl** [comment]
Ehr meh Gerhd that was a long one. LGTM with the indention change

**Comment by daviddahl** [comment]
Now I need to get my jslint act in order with my precious Emacs + js2-mode

File: server/test/stores/postgres/account.js (hunk 3 above)
Comment (grempe) [inline server/test/stores/postgres/account.js]: Not dumb.  I guess just trying to be consistent.  There is currently usage of both single and double quote styles.  I even tried to find if browsers favor one over the other.  I look at page 237 of the PDF linked in this article and it specifies that both are valid.

http://ejohn.org/blog/ecmascript-5-strict-mode-json-and-more/

So, no reason other than being pedantic I guess. :-)

**Comment by grempe** [comment]
Changed the indent to 2 spaces in .jshint and .editorconfig.  You should optimally add both jshint and editorconfig plugins to your editor and they will help ensure you are 'doing the right thing' TM.

File: server/test/stores/postgres/account.js (hunk 3 above)
Comment (daviddahl) [inline server/test/stores/postgres/account.js]: No worries. I don't care so much, @ecto did not like double quotes in JS code

**Comment by daviddahl** [comment]
:+1:

**Pull request merged**